docs/sql/statements/insert.rst

```diff
@@ -24,24 +24,31 @@ Synopsis
       { VALUES ( expression [, ...] ) [, ...] | ( query ) | query }
       [ ON CONFLICT (column_ident [, ...]) DO UPDATE SET { column_ident = expression [, ...] } |
         ON CONFLICT [ ( column_ident [, ...] ) ] DO NOTHING ]
+      [ RETURNING { * | output_expression [ [ AS ] output_name ] | relation.* } [, ...] ]
 
 Description
 ===========
 
-INSERT creates one or more rows specified by value expressions.
+``INSERT`` creates one or more rows specified by value expressions.
 
 The target column names can be listed in any order. If no list of column names
 is given at all, the default is all the columns of the table in lexical order;
```

Comment:
The `lexical order` part is actually outdated, we changed that behavior. We should follow up on this with a PR against master.